include/package-pack.mk

```diff
@@ -309,6 +309,20 @@ else
 	(cd $$(IDIR_$(1)) && find . -type f,l -printf "/%P\n" > $$(IDIR_$(1))/lib/apk/packages/$(1).list)
 	if [ -f $$(ADIR_$(1))/conffiles ]; then mv $$(ADIR_$(1))/conffiles $$(IDIR_$(1))/lib/apk/packages/$(1).conffiles; fi;
 
+	# Some package (base-files) manually append stuff to conffiles
+	# Append stuff from it and delete the CONTROL directory since everything else should be migrated
+	if [ -f $$(IDIR_$(1))/CONTROL/conffiles ]; then \
+		cat $$(IDIR_$(1))/CONTROL/conffiles >> $$(IDIR_$(1))/lib/apk/packages/$(1).conffiles; \
+		rm -rf $$(IDIR_$(1))/CONTROL/conffiles; \
+	fi
+
+	if [ -z "$$$$(ls -A $$(IDIR_$(1))/CONTROL 2>/dev/null)" ]; then \
+		rm -rf $$(IDIR_$(1))/CONTROL; \
+	else \
+		echo "CONTROL directory $$(IDIR_$(1))/CONTROL is not empty! This is not right and should be checked!" >&2; \
```

Comment:
This is cute but do we actually see those anywhere? I'd just go for `rm -rf CONTROL`

Reply:
I would like to keep it cause I have seen lots of strange thing done by package... It's a good way to catch them.

Reply:
Sure why not, please proceed.